src/components/tx/TxSimulation/index.tsx

```diff
@@ -86,5 +86,15 @@ export const TxSimulation = (props: TxSimulationProps): ReactElement | null => {
     return null
   }
 
+  if (!props.transactions) {
+    return (
+      <Box sx={{ mt: 2 }}>
```

Comment:
First of all, why do you need a box if you can just assign the margin to the skeletonWrapper? Secondly, for future: `<Box mt={2}`.

Reply:
I added the wrapper after the `<Box />`. thank you for the catch